design-reference/docs/azure-simple.md

```diff
@@ -0,0 +1,215 @@
+---
+title: Azure Simple
```

Comment:
Can this be inferred from the name of the folder hosting this file?

Reply:
It could, but I think it would be better not to, as this will be the "human readable" title seen in links. I can see cases in which users would want to title to be separate from the folder name.

Reply:
I was thinking that it could be generated by splitting on `-` and upper-casing the first letter of each word. May hold up better to refactors, though you definitely raise a good point. I'm OK either way!

Reply:
Hugo actually has a built-in "humanize" function that does this, so if we _do_ want to automate this, we won't need to create our own parser.